packages/webdriver/src/utils.js

```diff
@@ -117,13 +117,29 @@ export function getArgumentType (arg) {
 /**
  * creates the base prototype for the webdriver monad
  */
-export function getPrototype (isW3C, isChrome) {
+export function getPrototype (isW3C, isChrome, isMobile) {
```

Comment:
@christian-bromann Would it maybe make more sense to pass `capabilities` as an argument and determine `isMobile` and `isChrome` from within the getter?

Reply:
@martomo yeah, if we need to add one more `is*` as method param we have to rewrite it.